hackvote/app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def full_title(page_title = '')
+    base_title = ""
+    page_title.empty? ? base_title : page_title
```

Comment:
Is this conditional really necessary if `base_title` is always empty string, and page_title defaults to empty string?